how-bank-of-china-uses-a-scale-out-database-to-support-zabbix-monitoring-at-scale.md

```diff
@@ -0,0 +1,115 @@
+---
+title: 'How Bank of China Uses a Scale-Out Database to Support Zabbix Monitoring at Scale'
+author: ['Yu Han']
+date: 2020-10-12
+summary: To monitor their large-scale IT infrastructure, Bank of China chose TiDB, a scale-out relational database, to replace Zabbix's MySQL backend. Learn how they use TiDB to manage over 10,000 monitored devices and 18 TB data.
+tags: ['Scalability', 'MySQL compatibility', 'MySQL', 'Best practice']
+url: /case-studies/how-bank-of-china-uses-a-scale-out-database-to-support-zabbix-monitoring-at-scale/
+customer: Bank of China
+customerCategory: Financial Services
+categories: ['MySQL Scalability']
+logo: /images/blog/customers/bank-of-china-logo.png
+image: /images/blog/how-bank-of-china-uses-a-scale-out-database-to-support-zabbix-monitoring-at-scale.jpg
+---
+
+**Industry:** Banking
+
+**Author:** Yu Han (Engineer at Information Technology Operations Center, Bank of China)
+
+![How Bank of China Uses a Scale-Out Database to Support Zabbix Monitoring at Scale](media/how-bank-of-china-uses-a-scale-out-database-to-support-zabbix-monitoring-at-scale.jpg)
+
+Bank of China is the fourth largest state-owned commercial bank in China. Since 2016, we've been using Zabbix, a popular open-source monitoring solution, to monitor our IT infrastructure. We used to use MySQL as the backend storage for Zabbix; however, MySQL is not scalable enough to monitor IT environments on a large scale. After trying different solutions, we chose TiDB, a **MySQL-compatible**, **open-source**, **distributed SQL database** to replace MySQL as the backend for a large-scale Zabbix. (We call this collaboration **"TiZabbix"**.) With TiZabbix, we successfully monitor more than 10,000 hosts and query 18 TB of monitoring data.
```

Comment:
```suggestion
[Bank of China](https://en.wikipedia.org/wiki/Bank_of_China) is the fourth largest state-owned commercial bank in China. Since 2016, we've been using Zabbix, a popular open-source monitoring solution, to monitor our IT infrastructure. We used to use MySQL as the backend storage for Zabbix; however, MySQL is not scalable enough to monitor IT environments on a large scale. After trying different solutions, we chose TiDB, a **MySQL-compatible**, **open-source**, **distributed SQL database** to replace MySQL as the backend for a large-scale Zabbix. (We call this collaboration **"TiZabbix"**.) With TiZabbix, we successfully monitor more than 10,000 hosts and query 18 TB of monitoring data.
```